### PR TITLE
Avatar update

### LINKED ISSUE #89: Avatar update

### PR DESCRIPTION
**Pull request by n-s-mordvinov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "i18next": "21.3.2",
     "node-sass": "6.0.1",
     "react": "17.0.2",
+    "react-cropper": "^2.1.8",
```
[the remark this answers is not in the document]
Comment (sernik73) [reply package.json]: Done

File: src/form-helpers/utils.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+export const ContentTypes = {
+  JSON: 'application/json',
+  MPFD: 'multipart/form-data',
+}
+
+export const getBody = (data, type) => {
+  if (!data) return data;
+  switch (type) {
```
[the remark this answers is not in the document]
Comment (sernik73) [reply src/form-helpers/utils.js:23]: Done

File: src/components/profile/edit/form-item/index.jsx [hunk 3]
```diff
@@ -12,30 +12,37 @@ import style from './styles.module.scss';
...
-          :
-            <Input error={ meta.touched && meta.error } {...field} {...props} />
-        }
-        <div className={ style.formItemBtns }>
-          <Button onClick={ () => onCancel(name) }>{ t('formItem.cancel') }</Button>
-          <Button type='submit' primary>{ t('formItem.save') }</Button>
-        </div>
-      </div>
+      {
+        visivbleField === name ?
```
[the remark this answers is not in the document]
Comment (sernik73) [reply src/components/profile/edit/form-item/index.jsx:18]: Made a separate component

File: src/api/index.js [hunk 4]
```diff
@@ -42,6 +52,16 @@ export const fetchers = {
     const accessToken = await getToken();
     return axios.put(`/api/accounts/${id}`, data, { headers: {"Authorization" : `Bearer ${accessToken}`} });
   },
+  updateAvatarAccount: async ({id, data}) => {
+    const accessToken = await getToken();
```
[the remark this answers is not in the document]
Comment (sernik73) [reply src/api/index.js:56]: I'll think about how to simplify

**Review by kirillta — APPROVED** [review]

**Pull request merged**